src/runtime_src/xdp/profile/plugin/aie_trace_new/aie_trace_metadata.cpp

```diff
@@ -89,42 +77,33 @@ namespace xdp {
     auto counterScheme = xrt_core::config::get_aie_trace_counter_scheme();
 
     //Process the file dump interval
-    aie_trace_file_dump_int_s = xrt_core::config::get_aie_trace_file_dump_interval_s();
-    if (aie_trace_file_dump_int_s < MIN_TRACE_DUMP_INTERVAL_S){
+    aie_trace_file_dump_int_s = xrt_core::config::get_aie_trace_settings_file_dump_interval_s();
+    if (aie_trace_file_dump_int_s == DEFAULT_AIE_TRACE_DUMP_INTERVAL_S) {
+      // if set to default value, then check for old style config
+      aie_trace_file_dump_int_s = xrt_core::config::get_aie_trace_file_dump_interval_s();
+      if (aie_trace_file_dump_int_s != DEFAULT_AIE_TRACE_DUMP_INTERVAL_S)
+        xrt_core::message::send(xrt_core::message::severity_level::warning, "XRT",
+          "The xrt.ini flag \"aie_trace_file_dump_interval_s\" is deprecated and will be removed in future release. Please use \"file_dump_interval_s\" under \"AIE_trace_settings\" section.");
+    }
+    if (aie_trace_file_dump_int_s < MIN_TRACE_DUMP_INTERVAL_S) {
       aie_trace_file_dump_int_s = MIN_TRACE_DUMP_INTERVAL_S;
       xrt_core::message::send(xrt_core::message::severity_level::warning, "XRT", AIE_TRACE_DUMP_INTERVAL_WARN_MSG);
     }
 
     // Catch when compile-time trace is specified (e.g., --event-trace=functions)
     auto device = xrt_core::get_userpf_device(handle);
-    auto compilerOptions = get_aiecompiler_options(device.get());
-    runtimeMetrics = (compilerOptions.event_trace == "runtime");
+    // auto compilerOptions = get_aiecompiler_options(device.get());
+    // runtimeMetrics = (compilerOptions.event_trace == "runtime");
+    runtimeMetrics = true;
```

Comment:
I had a discussion with Jason on this. As "adf::aiecompiler_options" is not available in x86 flow, this check was removed. But, this will break "Edge" cases where runtime metrics is not used. The Edge flow needs to check and handle metrics appropriately. However, for the time being, this PR can be merged with this known issue. The issue can be fixed later. I am going to file a CR to track the issue.